### DISCUSSION TITLE
Get list of addresses using our plug&play project

### DISCUSSION THREAD

**Discussion by agurod42** [Web3Auth/Web3Auth#698] [Help]
Is there a way to get a list of all the addresses that have signed in using the plug&play module?

I can see the numbers on the dashboard, but the addresses info is not accesible.

**Comment by rinzler99** [comment]
Hey,
Plug&Play does not keep track of the list of all the addresses that signed in. As alternative, If you want to keep track of the signed addresses using plug&play I suggest you to call some api after user signs in and send the address of signed user to the backend.
